inigo_py/django/middleware.py

```diff
@@ -126,11 +128,7 @@ def __call__(self, request):
         response = self.get_response(request)
 
         # inigo: process response
-        processed_response = q.process_response(response.content)
-        if processed_response:
-            return self.respond(processed_response)
-
-        return response
+        return HttpResponse(q.process_response(response.content), status=200)
```

Comment:
content type header must be set if we switch from `JsonResponse` to `HttpResponse`. Otherwise it might be missing.